R/visualize.R

```diff
@@ -0,0 +1,154 @@
+#' Fairness Accuracy Tradeoff Visualization
+#'
+#' @description
+#' This function specialize in comparing the Fairness vs Accuracy between learners through visualizations.
+#' From the visualization, users could see the tradeoff between fairness metrics and accuracy.
+#' Those insights could help the users to choose the optimum model from their model sets.
+#' And the fairness consistency of the learners by visualizing the tradeoff for different resampling iterations.
+#' It could take multiple type of inputs, like (`Prediction()`) (`BenchmarkResult()`) and (`ResampleResult()`).
+#'
+#'
+#' @param predcitions (`Prediction()`) (`BenchmarkResult()`) (`ResampleResult()`)\cr
+#' The binary class prediction object that will be evaluated.
+#' * If provided as (`Prediction()`). Then only one point will indicate the accuracy and fairness metrics for the current predictions.
+#' * If provided as (`ResampleResult()`). Then the plot will compare the accuracy and fairness metrics for the same model, but different rasampling iterations.
+#' * If provided as (`BenchmarkResult()`). Then the plot will compare the accuracy and fairness metrics for all models and all rasampling iterations.
+#'
+#' @param fairness_measure (`Measure()`)\cr
+#' The fairness measures that will evaluated on predictions.
+#'
+#' @param data_task (`TaskClassif()`)\cr
```

Comment:
why not call it `task`?  data is redundant

Reply:
Removed data from data_task